tests/test_modeling_tf_common.py

```diff
@@ -321,10 +332,14 @@ def test_pt_tf_model_equivalence(self):
 
             # Check predictions on first output (logits/hidden-states) are close enought given low-level computational differences
             pt_model.eval()
-            pt_inputs_dict = dict(
-                (name, torch.from_numpy(key.numpy()).to(torch.long))
-                for name, key in self._prepare_for_class(inputs_dict, model_class).items()
-            )
+            pt_inputs_dict = {}
+            for name, key in self._prepare_for_class(inputs_dict, model_class).items():
+                if type(key) == bool:
+                    key = np.array(key, dtype=bool)
+                    pt_inputs_dict[name] = torch.from_numpy(key).to(torch.long)
```

Comment:
Any reason to change a bool to a tensor? We can just keep it as a boolean, no?

Reply:
This is because now, the `use_cache` parameter returned by the `prepare_inputs` method in the T5 test file is now a primitive boolean and not anymore a tensor. Thanks to the changes the value is not forced to be a tensor anymore :)

Reply:
Yes I understand that. PyTorch accepts regular booleans in its models, that's why I'm confused why we convert it to a PyTorch tensor here.

Reply:
This is because of the `.numpy()` to convert TF tensors to numpy array. Simple booleans don't have this attribute.

Reply:
My question is why not just pass `pt_inputs_dict[name] = key`

Reply:
Ah ok!! Simply because I didn't know 😄 Just pushed the update^^